style.css

```diff
@@ -89,3 +90,312 @@ header {
+  line-height: 24px;
+  text-align: left;
+  color: #091e42;
+  margin-bottom: 20px;
+}
+
+.recent-project-tech {
+  display: flex;
+  justify-content: space-evenly;
+  margin: 20px 0;
+  list-style: none;
+}
+
+.recent-project-tech-used {
+  border: 1px solid #babfc9;
+  padding: 5px 12px;
+}
+
+.see-recent-project {
+  background-color: #ff6b00;
+  color: #fff;
+  padding: 10px 20px;
+  border: none;
+}
+
+.see-recent-project:enabled {
+  color: #fff;
+  background-color: #f55800;
+}
+
+.see-recent-project:disabled {
+  color: #5b5957;
+  background-color: #f4f1f0;
+}
+
+.see-recent-project:hover:enabled {
+  color: #f55800;
+  background-color: #fff5e1;
+  border: 2px solid #f55800;
+}
+
+.see-recent-project:active:enabled {
+  background-color: #f55800;
+  color: #fff;
+}
+
+.projects-done {
+  margin-bottom: 20%;
+  background-image: url(./icons/Img-Placeholder.svg);
+  height: 100%;
+  background-repeat: no-repeat;
+  display: flex;
+  justify-content: flex-end;
+  flex-direction: column;
+  background-size: cover;
+  width: 100%;
+  padding: auto;
+}
+
+.my-recent-work {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  margin-bottom: 20px;
+}
+
+.project-note {
+  font-family: "Inter", sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  text-align: left;
+  color: #fff;
+  padding-left: 15px;
+}
+
+.project-stack-used {
+  display: flex;
+  list-style: none;
+  width: 80%;
+  justify-content: space-around;
+}
+
+.tech-used {
+  color: #fff;
+  font-family: "Inter", sans-serif;
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 20px;
+  padding: 8px;
+  margin: 10px 0;
+  background-color: #d1dbde63;
+}
+
+.see-project {
+  background-color: #ff6b00;
+  padding: 10px;
+  width: 100%;
+  text-align: center;
+  color: #fff;
+  border: none;
+  font-family: "Inter", sans-serif;
+  font-size: 17px;
+  font-weight: 700;
+  line-height: 24px;
+}
+
+.projects-card {
+  display: grid;
+  margin: 30px 10px;
+  gap: 25px;
+}
+
+.about-myself {
+  background-color: #1c1a19;
+}
+
+.about-me {
+  background: #1c1a19;
+  color: #fff;
+  background-image: url(./icons/Frame.png);
+  background-repeat: no-repeat;
+  background-position: bottom right;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  width: 100%;
+  justify-content: center;
+  padding: 0 20px;
+  height: 100vh;
+  margin-top: 50px;
+}
+
+.about-me-heading {
+  font-family: "Crete Round", serif;
+  color: #ff6b00;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+}
+
+.about-me-note {
+  font-family: "Inter", sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+  margin-top: 30px;
+}
+
+.get-my-resume {
+  font-family: "Inter", sans-serif;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  background-color: #ff6b00;
+  color: #fff;
+  padding: 15px;
+  margin-top: 20px;
+}
```

Comment:
- [ ] Kindly make sure to remove the border color which is appearing on your `Get resume` button in order to match the provided Figma design.

    - ### YOUR DESIGN
    
   ![image](https://user-images.githubusercontent.com/87197412/235523666-4debd8f7-d1d8-49da-8c3c-3c4ebd9e724e.png)

    - ### FIGMA DESIGN
    
    ![image](https://user-images.githubusercontent.com/87197412/235523370-ff2f1b41-0ad1-48af-a94d-6e26f888a874.png)